tests/unit/api/metadata/test_v0.py

```diff
@@ -964,3 +997,49 @@ def test_get_dashboard_bad_parameters(self) -> None:
             data = json.loads(response.data)
             self.assertEqual(response.status_code, HTTPStatus.INTERNAL_SERVER_ERROR)
             self.assertCountEqual(data.get('dashboard'), {})
+
+    @responses.activate
+    def test_get_related_dashboards_success(self) -> None:
+        """
+        Test get_related_dashboards API success
+        :return:
+        """
+        test_table = 'db://cluster.schema/table'
+        url = local_app.config['METADATASERVICE_BASE'] + TABLE_ENDPOINT + '/' + test_table + '/dashboard/'
+        responses.add(responses.GET, url, json=self.expected_related_dashboard_response, status=HTTPStatus.OK)
+
+        with local_app.test_client() as test:
+            response = test.get(
+                '/api/metadata/v0/table/{0}/dashboards'.format(test_table)
+            )
+            data = json.loads(response.data)
+            self.assertEqual(response.status_code, HTTPStatus.OK)
+            self.assertCountEqual(data.get('dashboards'), self.expected_related_dashboard_response)
```

Comment:
I believe this assertion will fail. Because our endpoint takes the response from metadata service and transforming each item slightly with `marshall_dashboard_partial`, at this point the result is going to be whatever the result of `[marshall_dashboard_partial(dashboard) for dashboard in self.expected_related_dashboard_response]` is.

Reply:
You are right! 
I couldn't make the count to work but len() did